vault/resource_token_test.go

```diff
@@ -188,6 +188,7 @@ resource "vault_token" "test" {
 }
 
 func TestResourceToken_expire(t *testing.T) {
+	t.Skip("skipping, because it's flaky in CI and there's a long time.Sleep call")
```

Comment:
I wonder if there is a better solution than just skipping? Or is it worth adding a bit of time to the tests so that we ensure we get this coverage in CI?

Reply:
I think it's worth skipping for now since these tests originally took 10-20seconds due to the `time.Sleep()` call. As part of VAULT-17253 I can try to rewrite this to avoid the Sleep call.